### PR TITLE
Added details of configuring the Umbraco application URL for licensing Workflow and Deploy.

### PR DESCRIPTION
**Pull request by AndyButland**
## Description

Following the addition of [this extra](https://github.com/umbraco/UmbracoDocs/pull/5417/) detail for licensing Umbraco Commerce, I've added similar for Workflow and Deploy.

@nathanwoulfe - can you review the changes for Workflow please?  I took out the part about setting the server role, as that won't be necessary with the latest version of the `Umbraco.Licenses` component.

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [X] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

Workflow 10, 11, 12 and Deploy 12

## Deadline (if relevant)

Can be reviewed and published at any time.

### CODE REVIEW

File: 12/umbraco-deploy/the-licensing-model.md
Comment (alina-tincas) [inline 12/umbraco-deploy/the-licensing-model.md:78]: seems the image is not working

**Review by alina-tincas — CHANGES_REQUESTED** [review]
Thank you for the PR @AndyButland #h5YR 🙌 

It seems one of the images is not working on the deploy licensing page. Otherwise it looks good and I will await an approval from @nathanwoulfe before merging this in 👍

File: 12/umbraco-deploy/the-licensing-model.md
Comment (AndyButland) [reply 12/umbraco-deploy/the-licensing-model.md:78]: It seems OK locally (i.e. I have the file at `\UmbracoDocs\12\umbraco-deploy\images\licenses-screen.png`).  And I've not actually added this in this PR.

**Comment by AndyButland** [comment]
Thanks, have fixed up.

**Review by alina-tincas — APPROVED** [review]

**Comment by alina-tincas** [comment]
All looks good! And just to confirm, for deploy `UmbracoApplicationUrl `is supposed to be only for v12?

**Comment by AndyButland** [comment]
That's correct yes, the new licensing model is only for Deploy 12+.

**Comment by alina-tincas** [comment]
Thank you for confirming. Then I am merging this in 🙌

**Pull request merged**
